Client/Frontend/Popover/PopoverContainerView.swift

```diff
@@ -115,7 +115,7 @@ extension PopoverController {
             super.layoutSubviews()
             
             // Assure the arrow will not be hanging off a corner
-            let clampedArrowXOrigin = min(max(arrowOrigin.x, PopoverUX.cornerRadius), bounds.width - PopoverUX.cornerRadius - PopoverUX.arrowSize.width / 2.0) - PopoverUX.arrowSize.width / 2.0
+            let clampedArrowXOrigin = min(max(arrowOrigin.x, PopoverUX.cornerRadius + 1), bounds.width - PopoverUX.cornerRadius - 1) - PopoverUX.arrowSize.width / 2.0
```

Comment:
What are those `1`s for? Maybe adding a comment if you feel it's important

Reply:
Mostly they're due to the fact that to make it look seamless we move the arrow down/up a pixel into the content (so theres no gap on rotation, etc.), so that means we have to offset the x as well by a pixel to avoid the arrow edges popping off the corner radius.

I'll add a comment